tests/blackbox/src/test_validation_data.py

```diff
@@ -0,0 +1,21 @@
+from validation.scenario_mock_responses import ValidationScenario
+from validation.utils.data_loader import load_data
+from validation.utils.utils import get_expectation
+
+# Unit tests run from the rootdir of the repository.
+DEFAULT_DATA_DIR: str = "tests/blackbox/data/namespace-scoped"
+
+
+def test_data_integrity():
```

Comment:
Can you add comment to explain the purpose of this test?

Reply:
will do